app/controllers/employees_controller.rb

```diff
@@ -0,0 +1,31 @@
+class EmployeesController < ApplicationController
+
```

Comment:
At this moment everyone can access this page shouldn't you protect it by adding `   before_action :authenticate_user!` or whatever session login that you're using